src/tabs/test/tabs.spec.js

```diff
@@ -98,6 +98,10 @@ describe('tabs', function() {
       titles().eq(1).find('> div').click();
       expect(scope.deselectFirst).toHaveBeenCalled();
     });
+
+    it('should expose controller as "tab" scope variable', function() {
+      expect(typeof elm.find('.uib-tab').first().isolateScope().tab).toEqual('object');
```

Comment:
I believe the only way to do this is:
```js
it(..., inject(function(uibtabDirective) {
  expect(elm.find('.uib-tab').first().isolateScope().tab).toBeInstanceOf(uibtabDirective[0].controller);
}));
```
Let me know if you are ok with such solution